int/docker-compose.yml

```diff
@@ -33,12 +34,24 @@ services:
       --auto-store-credential
       --preserve-exchange-records
       --public-invites
+    healthcheck:
+      test: curl -s -o /dev/null -w '%{http_code}' "http://localhost:3001/status/live" | grep "200" > /dev/null
+      start_period: 30s
+      interval: 7s
+      timeout: 5s
+      retries: 5
 
   echo:
     image: ghcr.io/indicio-tech/echo-agent:0.1.2
     ports:
       - "3002:3002"
     command: --host 0.0.0.0 --port 3002 --log-level debug
+    healthcheck:
+      test: nc -z localhost 3002
```

Comment:
I've found that on some version of the echo image (or something, I haven't nailed down exactly what the issue is), I've had to use `0.0.0.0` instead of `localhost` in the test command :thinking: